python/rikai/torch/data.py

```diff
@@ -167,7 +176,12 @@ def __init__(
         seed: Optional[int] = None,
         world_size: int = 1,
         rank: int = 0,
+        parquet_row_group_size_bytes: int = None
```

Comment:
maybe put this into either SparkSession.ctx or `rikai.option`? 

This is a writer site parameter, which should not on `Dataset`